src/routes/Group/components/TicketView.js

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import { Table } from 'semantic-ui-react';
+
+class TicketView extends React.Component {
+
+  actionRow(action){
+    return (
+      <tr>
+      <td>{action.actionTaken}</td>
+      <td>{action.timestamp}</td>
+      <td>{action.userId}</td>
+      </tr>
+    )
+  }
+
+  userLink(user) {
+    return user ? <a href={'/user/' + user._id +'/'}>{user.name}</a> : <span>No one</span>;
```

Comment:
Instead of normal anchors which reload the entire page, you should use `Link` components which take advantage of the router. See https://github.com/PagerNation/Escalator-UI/blob/master/src/routes/Home/components/GroupsInfo.js#L14